pkg/operator/vttablet/pod.go

```diff
@@ -92,6 +92,14 @@ func UpdatePod(obj *corev1.Pod, spec *Spec) {
 		vttabletAllFlags[key] = value
 	}
 
+	// Apply user-supplied extra flags last so they take precedence.
```

Comment:
This is already done right above.

Reply:
That was very silly of me - you're right.  Fixed https://github.com/planetscale/vitess-operator/pull/14/commits/676c06bac87aa48d59672de11d833a2258a7b3a3.